components/Home/WhatNow.tsx

```diff
@@ -1,9 +1,25 @@
+'use client';
 import React from 'react';
 import SectionTitle from '../Common/SectionTitle';
 import { useTranslations } from 'next-intl';
 
 function WhatNow() {
   const t = useTranslations('Components.home.whatNow');
+
+  const handleClick = (e: React.MouseEvent<HTMLAnchorElement>) => {
+    e.preventDefault();
+    const targetId = e.currentTarget.getAttribute('href');
+    if (targetId) {
+      const targetElement = document.querySelector(targetId) as HTMLElement; // Type assertion
+      if (targetElement) {
+        window.scrollTo({
+          top: targetElement.offsetTop,
+          behavior: 'smooth',
+        });
+      }
```

Comment:
You dont need to use the window object for this. 

You can use the targetElement (you know its an html elment and it exist): 

```js
targetElement.scrollIntoView({ behavior: 'smooth', block: 'start' });

```